test/e2e/i18n-data-fetching-redirect/app/pages/gsp-fallback-redirect/[param].js

```diff
@@ -0,0 +1,16 @@
+export async function getStaticProps() {
+  return {
+    redirect: {
+      destination: '/en/home',
```

Comment:
```suggestion
export async function getStaticProps({ locale }) {
  return {
    redirect: {
      destination: `/${locale}/home`,
```
Can we add a test case that ensures the locale passed down matches what we expect and it redirects correctly and then another test case that ensures redirecting from defaultLocale `en` -> `sv` works alright?